app/models/cookies.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class Cookies
+  include ActiveModel::Model
+  include ActiveModel::Attributes
+  include ActiveModel::Validations
+
+  attribute :analytics
```

Comment:
Is it possible for `analytics` to be a boolean rather than a string? It feels more natural to be using `true` and `false` rather than `'true'` and `'false'` in various places (eg, in `GoogleAnalytics::DataTracking#enabled?`)

I realise that it is being stored in a cookie. Is it possible to have a boolean stored in a cookie?

Reply:
I guess that the values of cookies have to be strings. I don't know if it helps but you can get a boolean value from a parameter (or cookie) that is a string using:

```ruby
ActiveModel::Type::Boolean.new.cast(params[:something])
```

(See https://api.rubyonrails.org/v6.1.0/classes/ActiveModel/Type/Boolean.html)

You could possibly use this in the `CookieConcern#set_datatracking_usage` method and then `usage_name` `GoogleAnalytics::DataTracking` would be a boolean.